webui/app/static/css/styles.css

```diff
@@ -4,4 +4,24 @@
 
 .txt-input {
     padding: 3px 7px;
-}
\ No newline at end of file
+}
+
+@media (max-width: 370px),
```

Comment:
I am trying to remember. Could you send me a screenshot of what these `@media` properties are doing? Or do they only apply if we uncomment out the options that we have hidden for now?